bentoml/_internal/bento/build_config.py

```diff
@@ -156,9 +274,52 @@ def write_to_bento(self, bento_fs: FS, build_ctx: str):
                 setup_script, bento_fs, docker_folder, "setup_script"
             )
 
+    @staticmethod
+    def validate_bentofile(yaml_content: t.Dict[str, t.Any]) -> None:
+        docker = yaml_content["docker"]
+
+        if "distro" in docker:
+            distro = docker["distro"]
+            if (
+                distro in ["alpine", "alpine-miniconda"]
+                and "scikit-learn" in yaml_content["python"]["packages"]
+            ):
+                sklearn_alpine_req = [
+                    "libquadmath",
+                    "musl",
+                    "lapack",
+                    "libgomp",
+                    "libstdc++",
+                    "lapack",
+                    "lapack-dev",
+                    "openblas-dev",
+                ]
+
+                if "system_packages" not in docker or docker["system_packages"] is None:
+                    raise_warning = True
+                    required_packages = sklearn_alpine_req
+                else:
+                    required_packages = list(
+                        set(sklearn_alpine_req).difference(docker["system_packages"])
+                    )
+                    raise_warning = len(required_packages) != len(sklearn_alpine_req)
+
+                if raise_warning:
+                    logger.warning(
+                        "In order to support [bold magenta]scikit-learn[/] on alpine-based images, "
+                        "make sure to include the following packages under "
+                        f"`docker.system_packages`: [bold yellow]{', '.join(required_packages)}[/]"
+                    )
+            if "conda" in yaml_content and distro not in CONDA_SUPPORTED_DISTRO:
+                raise BentoMLException(
+                    f"{distro} does not supports conda. Since 1.0.0a7, BentoML will "
```

Comment:
We probably don't need to mention the version here since it's from 1.0 onwards.

Reply:
done